harness/determined/exec/launch_autohorovod.py

```diff
@@ -21,6 +23,22 @@
 from determined.constants import HOROVOD_SSH_PORT
 
 
+def mask_config_dict(d: Dict) -> Dict:
+    mask = "********"
+    new_dict = copy.deepcopy(d)
+
+    # checkpoint_storage
+    hidden_checkpoint_stroage_keys = ["access_key", "secret_key"]
+    try:
+        for key in new_dict["checkpoint_storage"].keys():
+            if key in hidden_checkpoint_stroage_keys:
```

Comment:
two things: one is a spelling error ("stroage") and the other is that you could really just inline `["access_key", "secret_key"]` here like `if key in ("access_key", "secret_key"):`